src/mainwindow.cpp

```diff
@@ -977,7 +969,9 @@ void ConvertJsonToStringMap(const QJsonObject& valJson, const QStringList& field
         }
         else if (val.isString())
         {
-            nameValues[field] = "\"" + val.toString().replace("\n", " ").replace("\"", "\"\"") + "\"";
+            QString strVal = val.toString();
+            strVal.truncate(32000); // Excel cannot handle more than ~32K chars.
+            nameValues[field] = "\"" % strVal.replace("\n", " ").replace("\"", "\"\"") % "\"";
         }
         else if (val.isObject())
         {
```

Comment:
(this refers to line 1080 in method  CopyAllFiles, for some reason I can't put the comment directly there)
The "break" should be a "continue" instead; we want to skip the current iteration, not the whole loop. I got into a situation where I already had one of the twb file, but not the other. The loop broke and I didn't get the other file.